### ISSUE TITLE
How to set up the "Visual Studio Code" text editor for Python coding

### ISSUE THREAD

**Issue by cjrh** [BrisPy/talks#26]
Basic setup, tips, guidance etc.  Especially for managing virtualenv's.  Also, covering any differences between Windows/Linux/Mac would be good.  It might make sense to have multiple speakers coordinate, to cover any platform-specific issues.  I think that would be cool to rotate between three speakers, one for each platform.  

(I don't personally use vscode, but it's extremely popular and is a really good option for a smooth beginner experience)

**Comment by peterfi** [comment]
Is this topic still of interest?  I could probably do a session on this .. my experience is primarily on Windows, not Mac ..

**Comment by kaiw** [comment]
I think this would definitely be of interest to people. When might you be able to present? Our next meetup is tomorrow, and then November 13.

I don't think it matters that much to bring a Windows-focused experience here. Ideally it should mostly be cross-platform anyway!

**Comment by peterfi** [comment]
kai
Tomorrow (wed 9th Oct) clashes with a Azure meetup that I had planned to go to ,,, although your NLP topic is also of interest.  Sadly I'll be in NZ on holidays on the 13th November so not sure where that leaves us.  Happy to have a go next year if that's worthwhile

**Comment by kaiw** [comment]
Yeah, I think that's still definitely worthwhile! I'll mark this down as needing scheduling, and we should be able to figure something out in the new year.

**Comment by kaiw** [comment]
@peterfi Are you still interested in giving a talk on this? We have the beginner slot free for our February meetup (Wednesday 12th) if that would suit you.

**Comment by peterfi** [comment]
Kai

Sounds plausible .. Feb gives me enough time to develop something that
sounds like I know what I am talking about.

I might try and get to your Jan meeting on the 8th Jan but I could be away
but we'll see.

How long do you want me to confuse people for?

My mobile if you need it is 0414 987 192 and email is peterfi136@gmail.com

Regards
Peter Fitzsimon

**Comment by kaiw** [comment]
Excellent! I've penciled you in for the first slot in February (https://github.com/BrisPy/talks/blob/master/README.md). We usually aim for about 25 mins + 5 mins questions (give or take a few minutes... we're not that strict).
